layouts/shortcodes/youtube.html

```diff
@@ -0,0 +1,9 @@
+<div class="responsive-video">
```

Comment:
Your solution works great - thanks a lot for that!

I'm just wondering if we should model this shortcode override more closely on https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/shortcodes/youtube.html so that folks can still use the options as documented...?


```diff
--- /home/daniel/dev/website/hugo/tpl/tplimpl/embedded/templates/shortcodes/youtube.html	2021-09-13 13:58:27.103370314 +0200
+++ layouts/shortcodes/youtube.html	2022-05-16 11:19:33.660974736 +0200
@@ -2,9 +2,9 @@
 {{- if not $pc.Disable -}}
 {{- $ytHost := cond $pc.PrivacyEnhanced  "www.youtube-nocookie.com" "www.youtube.com" -}}
 {{- $id := .Get "id" | default (.Get 0) -}}
-{{- $class := .Get "class" | default (.Get 1) -}}
+{{- $class := .Get "class" | default "responsive-video" -}}
 {{- $title := .Get "title" | default "YouTube Video" }}
-<div {{ with $class }}class="{{ . }}"{{ else }}style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;"{{ end }}>
+<div class="{{ $class }}">
   <iframe src="https://{{ $ytHost }}/embed/{{ $id }}{{ with .Get "autoplay" }}{{ if eq . "true" }}?autoplay=1{{ end }}{{ end }}" {{ if not $class }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}allowfullscreen title="{{ $title }}"></iframe>
 </div>
 {{ end -}}
```

Not sure if anyone else feels strongly about this...?